addons/l10n_se/data/account.account.template.csv

```diff
@@ -272,15 +262,14 @@
 "a7324","7324","Skattepliktiga traktamenten, utlandet","account.data_account_type_expenses","l10nse_chart_template","False"
 "a7331","7331","Skattefria bilersättningar","account.data_account_type_expenses","l10nse_chart_template","False"
 "a7332","7332","Skattepliktiga bilersättningar","account.data_account_type_expenses","l10nse_chart_template","False"
-"a7333","7333","Ersättning för trängselskatt, skattefri","account.data_account_type_expenses","l10nse_chart_template","False"
```

Comment:
It's not a good idea to include accounts outside minimum, or the commonly used accounts in sweden.

The list would be extensive. The complete list would include several hundreds of accounts most not used by normal companies.

There exist flavors for different business types, Limited. Sole trader Associations etc.

The most commonly used is AB (Limited mid sized companies).
We only have this kind of companies in odoo today.

I think is better to add accounts for other kind of business upon need, you are free to amend the extra ones still following BAS.

Most companies would have a fit o 95-99% only using the  "Minimum".
Probably most feels a need to add some extra :-). 
There are also some commonly used in several accounting software

BAS website is a swedish stakholder assostation (https://www.bas.se/english/)
Most buisinesses are following this. Bigger and global companies may do different.
All of the swedish accounting systems are represented in this association

In the Swedish version the commonly used is dot marked. ( refered to as Minimum above )

To add additional accounts it may be good for a single user, the association has marked i "commonly used"
To use another year release could be possibly be argued.

So Its a commonly agreed vesion that is used by most companies to add extra account don't seems wise to me.

We both have removed most of the accounts not used by Limited companies, I guess for a reason ?

odoo is auto creating the bank accounts ranging 1910-1949 (in BAS). 
We have noted its not good to include this, better to let odoo do the work

But its better odoo decides whats best.

Regarding the rest of the accounts i don't think it's good for anyone to amend the standard
I beileve BAS has already done so, i would not arue what is a good or bad choise  and we can all have different opinion.

Reply:
@jco-odoo any comment on this ?

Reply:
Better to stay to the minimum accounts and it is true Odoo will create bank accounts automatically, so the accounts might create confusion.  (need to check from a user perspective)

Reply:
@jco-odoo when can we merge this in the main ? I think the issues has been discussed and are solved